core/src/cubos/core/gl/ogl_render_device.cpp

```diff
@@ -14,6 +14,24 @@ using namespace cubos::core::gl;
 
 #define LOG_GL_ERROR(err) CUBOS_ERROR("OpenGL error: {}", err)
 
+// OpenGL message callback (errors, warnings, etc)
+static void GLAPIENTRY messageCallback(GLenum source, GLenum type, GLuint id, GLenum severity, GLsizei length,
+                                       const GLchar* message, const void* userParam)
+{
+    (void)length;
+    (void)userParam;
+    (void)id;
+
+    if (severity == GL_DEBUG_SEVERITY_HIGH)
+    {
+        CUBOS_ERROR("OpenGL message (source = 0x{:x}, type = 0x{:x}): {}", source, type, message);
+    }
+    else if (severity == GL_DEBUG_SEVERITY_MEDIUM)
+    {
+        CUBOS_WARN("OpenGL message (source = 0x{:x}, type = 0x{:x}): {}", source, type, message);
```

Comment:
Our errors starting with "OpenGL error:" but errors and warnings emitted by Opengl starting with "OpenGL message" is confusing